README.md

```diff
@@ -20,6 +20,7 @@
 # Apache Arrow
 
 [![Build Status](https://travis-ci.org/apache/arrow.svg?branch=master)](https://travis-ci.org/apache/arrow)
+[![Build Status](https://ci.appveyor.com/api/projects/status/01dv4jp5tc59ked3/branch/master?svg=true)](https://ci.appveyor.com/project/ApacheSoftwareFoundation/arrow/branch/master)
```

Comment:
I think this should probably use the [public repo link style](https://www.appveyor.com/docs/status-badges/#badges-for-projects-with-public-repositories-on-github-and-bitbucket) which is more readable:
```suggestion
[![Build Status](https://ci.appveyor.com/api/projects/status/github/apache/arrow/branch/master?svg=true)
```

Reply:
Done.